docs/use/gateway.md

```diff
@@ -41,6 +41,11 @@ Auto discovery is enable by default on release binaries, on platformio (except f
 If the new connection fails the gateway will fallback to the previous connection.
 :::
 
+Define 'MQTTsetWIFI' to activate this
+```
+build_flags = '-DMQTTsetWIFI'
+```
+
```

Comment:
I think we can remove this, this function is not impacting a lot the program size